features/steps/thens/attributes.py

```diff
@@ -109,3 +113,18 @@ def step_impl(context, attribute, value):
                 yield(err.InvalidValueError(False, inst, attribute, attribute_value))
             elif context.error_on_passed_rule:
                 yield(err.RuleSuccessInst(True, inst))
+
+
+@then('The {field} of the {model_or_field} must be "{values}"')
+@err.handle_errors
+def step_impl(context, field, model_or_field, values):
+    values = misc.strip_split(values, strp='"', splt=' or ')
+    model_schema = context.model.schema_identifier.lower()
+    if field == "Schema Identifier":
+        s = context.model.schema_identifier
+        if not s.lower() in values:
+            yield err.IncorrectSchemaError(False, s, values)
+    elif field == "Schema" and not context.model.schema in values:
```

Comment:
Perhaps add an else statement with RulePassed? Does it make sense for this particular rule?

Reply:
Maybe not, but let's be consistent. 
https://github.com/buildingSMART/ifc-gherkin-rules/pull/114/commits/26365972fa5a68a6b7012d077dcbc0ab9a1ae337

There's a 'weird' caveat in this rule. That is, all the other rules so far work with `context.instances`. But this one doesn't necessarily. I took `Given An IfcProject` to match the logic of our fail/success implementation. But maybe it's too risky to assume there is always an IfcProject.